File: src/Domain/Models/Answer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class Answer
    {
        public int Id { get; set; }
        public string Ans { get; set; }
        public int Value { get; set; }
        public int QuestionId { get; set; }
    }
}

File: src/SurveySays/wwwroot/ngApp/models/answer.ts
namespace SurveySays.Models {

    export interface IAnswer {
        id: number;
        ans: string;
        value: number;
        questionId: number;
    }

    export class Answer implements IAnswer {
        id: number;
        ans: string;
        value: number;
        questionId: number;
    }
}